dulwich/tests/__init__.py

```diff
@@ -112,41 +112,7 @@ def run_command(self, name, args):
 
 def self_test_suite():
     names = [
-        "archive",
```

Comment:
Uhm, is this just GitHub on mobile acting up or does this remove the list of test modules?

Reply:
Oops, didn't mean to commit that!